src/taco-cli/test/kit.ts

```diff
@@ -214,67 +213,67 @@ describe("Kit command : ", function (): void {
 
         // Force KitHelper to fetch the package fresh
         kitHelper.kitPackagePromise = null;
-
-        this.timeout(60000);
         rimraf.sync(runFolder);
     });
 
-    after((done: MochaDone) => {
+    after() => {
         process.env["TACO_UNIT_TEST"] = previous;
         process.chdir(originalCwd);
         kitHelper.kitPackagePromise = null;
-        rimraf(runFolder, function (err: Error): void { done(); }); // ignore errors
+        rimraf.sync(runFolder); // ignore errors
     });
 
-    it("'taco kit' should not throw any error", function (done: MochaDone): void {
-        kitRun()
-            .done((telemetryParameters: TacoUtility.ICommandTelemetryProperties) => {
-                var expected: TacoUtility.ICommandTelemetryProperties = { subCommand: { isPii: false, value: "list" } };
-                telemetryParameters.should.be.eql(expected);
-                done();
-            }, function (err: tacoUtils.TacoError): void {
-                done(err);
-            });
-    });
+    describe("Kit list command : ", function (): void {
+        it("'taco kit' should not throw any error", function (done: MochaDone): void {
+            kitRun()
+                .done((telemetryParameters: TacoUtility.ICommandTelemetryProperties) => {
```

Comment:
Looks like we don't need the done handler, so we can change this to just return kitRun().then(...)